x/pairing/types/key_badge_used_cu.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import "encoding/binary"
+
+var _ binary.ByteOrder
+
+const (
+	// BadgeUsedCuKeyPrefix is the prefix to retrieve all BadgeUsedCu
+	BadgeUsedCuKeyPrefix  = "BadgeUsedCu/value/"
+	BadgeTimerStorePrefix = "BadgeTimerStore/"
+)
+
+// BadgeUsedCuKey returns the store key to retrieve a BadgeUsedCu from the index fields
+func BadgeUsedCuKey(
+	badgeUsedCuMapKey []byte,
+) []byte {
+	var key []byte
+
+	key = append(key, badgeUsedCuMapKey...)
```

Comment:
```return append(badgeUsedCuMapKey, byte('/'))```
but I suggest to do that when generating the badge key (once) and not upon every access to the store.